src/slice.cr

```diff
@@ -545,6 +546,106 @@ struct Slice(T)
     @pointer
   end
 
+  # Returns a new slice with all elements sorted based on the return value of
+  # their comparison method `#<=>`
+  #
+  # ```
+  # a = Slice[3, 1, 2]
+  # a.sort # => Slice[1, 2, 3]
+  # a      # => Slice[3, 1, 2]
+  # ```
+  def sort : Slice(T)
+    dup.sort!
+  end
+
+  # Returns a new slice with all elements sorted based on the comparator in the
+  # given block.
+  #
+  # The block must implement a comparison between two elements *a* and *b*,
```

Comment:
*This* style is usually used for arguments. But there's no argument named `a` or `b` here.
```suggestion
  # The block must implement a comparison between two elements a and b,
```